src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -63,26 +63,26 @@ namespace
     QJsonValue triStateBoolToJsonValue(const TriStateBool &triStateBool)
     {
         switch (static_cast<int>(triStateBool)) {
-        case 0:  return false; break;
-        case 1:  return true; break;
+        case 0:  return false;
+        case 1:  return true;
         default: return QJsonValue();
         }
     }
 
     TriStateBool addPausedLegacyToTriStateBool(int val)
     {
         switch (val) {
-        case 1:  return TriStateBool::True; break; // always
-        case 2:  return TriStateBool::False; break; // never
+        case 1:  return TriStateBool::True; // always
+        case 2:  return TriStateBool::False; // never
         default: return TriStateBool::Undefined; // default
         }
     }
 
     int triStateBoolToAddPausedLegacy(const TriStateBool &triStateBool)
     {
         switch (static_cast<int>(triStateBool)) {
-        case 0:  return 2; break; // never
-        case 1:  return 1; break; // always
+        case 0:  return 2; // never
+        case 1:  return 1; // always
         default: return 0; // default
         }
     }
```

Comment:
IMO, these changes belong to the previous commit.

Reply:
Ok. Moved there.